api/src/main/kotlin/edu/wgu/osmt/keyword/KeywordController.kt

```diff
@@ -1,278 +1,129 @@
-    }
+    fun keywordById(
+        @PathVariable id: Long,
+    ): HttpEntity<ApiKeyword> {
 
-    @GetMapping(RoutePaths.CATEGORY_SKILLS, produces = [MediaType.APPLICATION_JSON_VALUE])
-    @ResponseBody
-    fun getCategorySkills (
-        uriComponentsBuilder: UriComponentsBuilder,
-        @PathVariable identifier: String,
-        @RequestParam(required = false, defaultValue = PaginationDefaults.size.toString()) size: Int,
-        @RequestParam(required = false, defaultValue = "0") from: Int,
-        @RequestParam(
-            required = false,
-            defaultValue = PublishStatus.DEFAULT_API_PUBLISH_STATUS_SET
-        ) status: Array<String>,
-        @RequestParam(required = false) sort: String?,
-        @AuthenticationPrincipal user: Jwt? = null
-    ): HttpEntity<List<RichSkillDoc>> {
-        return searchCategorySkills(
-            uriComponentsBuilder = uriComponentsBuilder,
-            identifier = identifier,
-            size = size,
-            from = from,
-            status = status,
-            sort = sort,
-            user = user,
-        )
+        return ResponseEntity
+            .status(HttpStatus.OK)
+            .body(this.byId(id) ?: throw ResponseStatusException(HttpStatus.NOT_FOUND))
     }
 
-    @PostMapping(RoutePaths.CATEGORY_SKILLS, produces = [MediaType.APPLICATION_JSON_VALUE])
+    @PostMapping(RoutePaths.KEYWORD_CREATE, produces = [MediaType.APPLICATION_JSON_VALUE])
     @ResponseBody
-    fun searchCategorySkills (
-        uriComponentsBuilder: UriComponentsBuilder,
-        @PathVariable identifier: String,
-        @RequestParam(required = false, defaultValue = PaginationDefaults.size.toString()) size: Int,
-        @RequestParam(required = false, defaultValue = "0") from: Int,
-        @RequestParam(
-            required = false,
-            defaultValue = PublishStatus.DEFAULT_API_PUBLISH_STATUS_SET
-        ) status: Array<String>,
-        @RequestParam(required = false) sort: String? = null,
-        @RequestBody(required = false) apiSearch: ApiSearch? = null,
-        @AuthenticationPrincipal user: Jwt? = null
-    ): HttpEntity<List<RichSkillDoc>> {
-        val sortEnum = sort?.let{ SkillSortEnum.forApiValue(it)}
-
-        val id: Long = identifier.toLong()
-        val keyword = keywordRepository.findById(id)
-
-        if (keyword?.type != KeywordTypeEnum.Category) throw ResponseStatusException(HttpStatus.NOT_FOUND)
-
-        return searchRelatedSkills(
-            uriComponentsBuilder = uriComponentsBuilder,
-            keyword = keyword,
-            size = size,
-            from = from,
-            statusFilters = status,
-            sort = sortEnum ?: SkillSortEnum.defaultSort,
-            apiSearch = apiSearch ?: ApiSearch(),
-            user = user
-        )
+    @PreAuthorize("hasAuthority(@appConfig.roleAdmin)")
+    fun createKeyword(
+        @RequestBody apiKeywordUpdate: ApiKeywordUpdate,
```

Comment:
According to open API, this should receive a list of keywords, not a single keyword.